### ISSUE TITLE
JAWS reads out each result as 1 of x

### ISSUE THREAD

**Issue by accessiblewebuk** [alphagov/accessible-autocomplete#367]
For example, typing in "UN" produces 11 results. Scrolling down the list, each one is read out as "1 of 11" - although it is one of the optional acceptance criteria to read out the number of results, if it is implemented then it needs to indicate the correct number.

**Comment by nickcolley** [comment]
I believe this is solved in v2.0.0 but will need to double check.

**Comment by vanitabarrett** [comment]
Tested with JAWS + latest versions of Chrome, Edge, Firefox and IE11 and was unable to reproduce the error. Going to close this as it looks like it's been fixed.
